### PR TITLE
docs: add async main and test charter description

### PR DESCRIPTION
**Pull request by vincenzopalazzo**
This adds a landing page and marks the work as started.

I will work on a blog post to discuss together the milestone to add, do this job in sync I think will benefit to collect more ideas from people that are doing work where this initiative depends on.

Signed-off-by: Vincenzo Palazzo <vincenzopalazzodev@gmail.com>

### CODE REVIEW

File: src/vision/roadmap.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ Clicking on active initiatives also shows a list of *milestones*. These mileston
 [fundamentals]: https://rust-lang.github.io/async-fundamentals-initiative/
 [Async closures]: https://rust-lang.github.io/async-fundamentals-initiative/design-discussions/async_closures.html
 [Boxable async functions]: ./roadmap/async_fn/boxable.md
-[Async main and tests]: ./roadmap/async_fn/async_main_and_tests.md
+[Async main and tests]: https://rust-lang.github.io/wg-async
```
Comment (eholk) [inline src/vision/roadmap.md:72]: This doesn't look like the right link?

**Comment by eholk** [comment]
I think the landing page might be missing from this PR...

File: src/vision/roadmap.md (hunk 1 above)
Comment (vincenzopalazzo) [reply src/vision/roadmap.md]: Yeah sorry, I may copy and paste the wrong link, it is this one https://vincenzopalazzo.github.io/async-main-and-tests-initiative/index.html and now should be included inside the change

**Comment by vincenzopalazzo** [comment]
>I think the landing page might be missing from this PR...

Thanks for the review @eholk and sorry for the mistake, now it should contain the pointer to the [landing page](https://vincenzopalazzo.github.io/async-main-and-tests-initiative/index.html)

**Comment by eholk** [comment]
Thanks! At some point we should probably look into pulling your repo under the rust-lang org like the other wg-async initiatives.

**Pull request merged**